### DISCUSSION TITLE
The migration plan for `parse_expr`

### DISCUSSION THREAD

**Discussion by Xuanwo** [databendlabs/databend#7073] [Q&A] [answered]
I'm trying to remove the old planner from the databend. But I'm blocked by the depends on `parse_expr`:

https://github.com/datafuselabs/databend/blob/dc289e0619daa95dfb7795817c32d6c8985b2db2/query/src/sql/sql_parser.rs#L87-L89

What are the alternative functions in the new planner? Used in the following places:

https://github.com/datafuselabs/databend/blob/dc289e0619daa95dfb7795817c32d6c8985b2db2/query/src/interpreters/interpreter_table_show_create.rs#L52-L139

cc @andylokandy @leiysky

**Comment by andylokandy** [comment, marked as answer]
Do you want this one https://github.com/datafuselabs/databend/blob/a7f2c587d522cab88db9171f01e583ae9e8bb0b5/common/ast/src/parser/mod.rs#L60

**Reply by Xuanwo** [reply]
I tried but failed for missing this call:

```rust
format!(" DEFAULT {}", expression.column_name())
```

**Reply by leiysky** [reply]
The equivalent approach is `format!(" DEFAULT {:#}", expr)`
